### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by louridas**
Mostly typos and matters of style.

### CODE REVIEW

File: hcert_spec.md [hunk 1]
```diff
@@ -65,51 +65,51 @@ The integrity and authenticity of origin of payload data MUST be verifiable by t
...
 
-Due to the shortening of the identifier (for space-preserving reasons) there is a slim but non-finite chance that the overall list of DSCs accepted by a validator may contain DSCs with duplicate KIDs. For this reason a verifier MUST check all DSCs with that KID.
+Due to the shortening of the identifier (for space-preserving reasons) there is a slim but non-finite chance that the overall list of DSCs accepted by a validator may contain DSCs with duplicate `kid`s. For this reason a verifier MUST check all DSCs with that `kid`.
 
 ####  Issuer
 
-The Issuer (**iss**) claim is a string value which MAY optionally hold the ISO 3166-1 alpha-2 Country Code of the entity issuing the health certificate. This claim can be used by a Verifier to identify which set of DSCs to use for validation. The Claim Key 1 is used to identify this claim.
+The Issuer (`iss`) claim is a string value thAT MAY optionally hold the ISO 3166-1 alpha-2 Country Code of the entity issuing the health certificate. This claim can be used by a Verifier to identify which set of DSCs to use for validation. The Claim Key 1 is used to identify this claim.
```
Comment (SZoerner) [inline hcert_spec.md:92]:
```suggestion
The Issuer (`iss`) claim is a string value that MAY optionally hold the ISO 3166-1 alpha-2 Country Code of the entity issuing the health certificate. This claim can be used by a Verifier to identify which set of DSCs to use for validation. The Claim Key 1 is used to identify this claim.
```

File: hcert_spec.md [hunk 2]
```diff
@@ -196,42 +195,42 @@ To mitigate against the risks that the signing algorithm is found to be weak, al
...
+## The Key Identifier (`kid`s)
 
-The key identifier (KID) is calculated when constructing the list of trusted public keys from DSC certificates and consists of a truncated (first 8 bytes) SHA-256 fingerprint of the DSC encoded in DER (raw) format.
+The key identifier (`kid`) is calculated when constructing the list of trusted public keys from DSC certificates and consists of a truncated (first 8 bytes) SHA-256 fingerprint of the DSC encoded in DER (raw) format.
 
-Note that Verifiers do not need to calculate the KID based on the DSC certificate and can directly match the key identifier in issued health certificate with the KID on the trusted list.
+Note that Verifiers do not need to calculate the `kid` based on the DSC certificate and can directly match the key identifier in issued health certificate with the `kid` on the trusted list.
 
-## Differences to the ICAO MasterList system for passports
+## Differences with the ICAO MasterList System for Passports
```
Comment (SZoerner) [inline hcert_spec.md:233]:
```suggestion
## Differences to the ICAO MasterList System for Passports
```

File: hcert_spec.md [hunk 3]
```diff
@@ -242,31 +241,31 @@ While patterned on best practices of the ICAO ML, there are a number of simplifi
...
 
-**In all cases - the secretariat acts not as content owner, all signatures and certificates must be provided by attendees.**
+**In all cases, the secretariat acts not as content owner, all signatures and certificates must be provided by attendees.**
 
-Note 1: The curent DGCG design allows for list to be up to date in real time (i.e. it is dynamically generated from a database with the most up to date information then available. So the here aggregated is the output of a data query to given parameters against the uploaded data by the memberstates) The technical requirement is lighter - the lists should be updated within 24 hours of any change submitted by a memberstate.
+Note 1: The curent DGCG design allows for list to be up to date in real time (i.e., it is dynamically generated from a database with the most up to date information then available. So the here aggregated is the output of a data query to given parameters against the uploaded data by the member states) The technical requirement is lighter - the lists should be updated within 24 hours of any change submitted by a member state.
```
Comment (SZoerner) [inline hcert_spec.md:259]:
```suggestion
Note 1: The current DGCG design allows for the list to be up to date in real-time (i.e., it is dynamically generated from a database with the most up to date information available at this point in time. So the here aggregated list is the output of a data query to given parameters against the uploaded data by the Member States). The technical requirement is lighter - the lists should be updated within 24 hours of any change submitted by a Member State.
```

**Review by SZoerner — COMMENTED** [review]
Minor typo, case consistency.

**Review by fredriklj — APPROVED** [review]

**Pull request merged**